tflow2.js

```diff
@@ -19,66 +19,20 @@ const OpEqual = 0;
 const OpUnequal = 1;
 const OpSmaller = 2;
 const OpGreater = 3;
-const FieldTimestamp = 0;
-const FieldRouter = 1;
-const FieldSrcAddr = 2;
-const FieldDstAddr = 3;
-const FieldProtocol = 4;
-const FieldIntIn = 5;
-const FieldIntOut = 6;
-const FieldNextHop = 7;
-const FieldSrcAs = 8;
-const FieldDstAs = 9;
-const FieldNextHopAs = 10;
-const FieldSrcPfx = 11;
-const FieldDstPfx = 12;
-const FieldSrcPort = 13;
-const FieldDstPort = 14;
-const fields = {
-        "Router": 1,
-        "SrcAddr": 2,
-        "DstAddr": 3,
-        "Protocol": 4,
-        "IntIn": 5,
-        "IntOut": 6,
-        "NextHop": 7,
-        "SrcAsn": 8,
-        "DstAsn": 9,
-        "NextHopAsn": 10,
-        "SrcPfx": 11,
-        "DstPfx": 12,
-        "SrcPort": 13,
-        "DstPort": 14,
-};
-const fieldById = {
-    "1": "Router",
-    "2": "SrcAddr",
-    "3": "DstAddr",
-    "4": "Protocol",
-    "5": "IntIn",
-    "6": "IntOut",
-    "7": "NextHop",
-    "8": "SrcAsn",
-    "9": "DstAsn",
-    "10": "NextHopAsn",
-    "11": "SrcPfx",
-    "12": "DstPfx",
-    "13": "SrcPort",
-    "14": "DstPort"
-};
 
 var bdfields = [
         "SrcAddr", "DstAddr", "Protocol", "IntIn", "IntOut", "NextHop", "SrcAsn", "DstAsn",
         "NextHopAsn", "SrcPfx", "DstPfx", "SrcPort", "DstPort" ];
 
+var fields = ["Router"].concat(bdfields)
+
 function drawChart() {
     var query = $("#query").val();
     if (query == "" || query == "{}") {
         return;
     }
 
     var url = "/query?q=" + encodeURI(query)
-    console.log(url);
```

Comment:
It breaks the script if `console` does not exist. The developer tools of your brokers already track all AJAX requests.